common/src/device/transport/mqtt/ClientWebSocketChannel.cs

```diff
@@ -286,7 +285,7 @@ public override bool Equals(object obj)
                 return true;
             }
 
-            if (ReferenceEquals(obj, null))
+            if (obj is null)
```

Comment:
is is usually used for type comparison. Why not obj == null?

Reply:
Good point; for `null` is resolves to the same result; so I used the `is` operator. We are also overriding the `==` operator below, so I didn't want to depend on it for this comparison.

Reply:
https://www.gullberg.tk/blog/is-null-versus-null-in-c/

Reply:
But I agree, I'm used to seeing `== null` checks

Reply:
Yeah I saw that the == is overloaded below. The logic was confusing in the beginning but I think I get it now.

Reply:
For null, using == will surely resolve to the same even if you have an overload.